tests/e2e/flowtest/test_scc.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import logging
+
```

Comment:
IIUC, we are creating a customer SCC and re-spinning the pods, should we change the test script name reflecting the same to make it more clear?

Reply:
makes sense! its hard to come up with good names :p